index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      header__buttoncontent="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20,500,0,-25"
+    />
+    <title>Medical core</title>
+  </head>
+  <body>
+    <!-- start navbar -->
+    <nav class="header">
+      <div class="menu">
+        <div class="header__icon">
+          <h1 class="header__name">Medical core</h1>
+        </div>
+        <ul class="header__links">
+          <li class="header_link"><a href="#">Home</a></li>
```

Comment:
I don't know if  `header_links` is block but if it was a block you can use a single hyphen. if it was block element you can use two underscores